Crashlytics/Crashlytics/Controllers/FIRCLSReportUploader.m

```diff
@@ -94,7 +94,9 @@ - (void)prepareAndSubmitReport:(FIRCLSInternalReport *)report
         dispatch_once(&regenerateOnceToken, ^{
           // Check to see if the FID has rotated before we construct the payload
           // so that the payload has an updated value.
-          [self.installIDModel regenerateInstallIDIfNeeded];
+          [self.installIDModel regenerateInstallIDIfNeededWithBlock:^(NSString *_Nonnull newFIID) {
+            self.fiid = [newFIID copy];
```

Comment:
Why explicitly copy when the property is already qualified as "Copy"?

Reply:
It's kindof a funny thing, but the style guide suggests you copy strings when you don't know where they came from: https://google.github.io/styleguide/objcguide.html#setters-copy-nsstrings. The main thing is to avoid cases of it being mutable under the hood or having different retain semantics than what you expect.

Reply:
Interesting. Surprisingly their example is a on an override setter method which makes it even more weird. Since there is no harm in re-copying, I'm fine with this change.